src/juicy-popover.html

```diff
@@ -209,93 +103,201 @@
 
             return newExpandableRect;      
         }
+            
+        class JuicyPopoverElement extends HTMLElement {
+            
+            get disabled(){
+                return this._disabled;
+            }
+            
+            set disabled(val) {
+                if (this._disabled !== val) {
+                    this._disabled = val;
+                    if (val) {
+                        this._expanded = false;
+                        this.setAttribute('disabled', '');
+                    }
+                    else {
+                        this.removeAttribute('disabled');
+                    }
+                }
+            }
+            
+            get expanded(){
+                return this._expanded;
+            }
+            
+            set expanded(val) {
+                if (this._expanded !== val) {
+                    this._expanded = val;
+                    if(val) {
+                        this.setAttribute('expanded', '');
+                        this._expand();
+                    } else {
+                        this.removeAttribute('expanded');
+                        this._collapse();
+                    }
+                }
+            }
+                
+            constructor(self) {
+                // Polyfill ceveat we need to fetch the right context;
+                // https://github.com/WebReflection/document-register-element/tree/master#v1-caveat
+                self = super(self);
+                // Creates the shadow root
+                ShadyCSS.styleElement(self);
```

Comment:
Same as above, what if there is no `ShadyCSS`, also here you are assuming Shadow DOM v1 polyfill, while lines bellow support V1 and V0.